Documentation/helm-operator.md

```diff
@@ -73,8 +77,8 @@ helm install --namespace rook-ceph rook-master/rook-ceph --version v0.7.0-278.gc
 To deploy from a local build from your development environment:
 
 1. Build the Rook docker image: `make`
-1. Copy the image to your K8s cluster, such as with the `docker save` then the `docker load` commands
-1. Install the helm chart:
+2. Copy the image to your K8s cluster, such as with the `docker save` then the `docker load` commands
```

Comment:
The markdown rendered takes care of the numbers.
E.g., this allows any point to be moved around without having to update the numbers of the points all the time.

Please leave the points as "1."